src/main/java/fr/openent/viescolaire/service/impl/DefaultInitService.java

```diff
@@ -229,4 +228,31 @@ public Future<JsonObject> initServices(String structureId, SubjectModel subject)
 
         return promise.future();
     }
+
+    @Override
+    public Future<JsonObject> initExclusionPeriod(String structureId, String zone) {
+        Promise<JsonObject> promise = Promise.promise();
+
+        if (!"A".equals(zone) && !"B".equals(zone) && !"C".equals(zone)) {
+           promise.complete();
+        } else {
+            JsonObject action = new JsonObject()
+                    .put(Field.ACTION, "init")
+                    .put(Field.STRUCTUREID, structureId)
+                    .put(Field.ZONE, zone)
+                    .put(Field.INITSCHOOLYEAR, false);
+
+            eb.request(EDT_ADDRESS, action, res -> {
+                if (res.failed()) {
+                    LOGGER.error(String.format("[Viescolaire@%s::initExclusionPeriod] Failed to init exclusion period",
```

Comment:
on aura pas l'exception exact, faudrait inclure ton res.cause() dans ton string.format

```suggestion
 LOGGER.error(String.format("[Viescolaire@%s::initExclusionPeriod] Failed to init exclusion period : %s",
                            this.getClass().getSimpleName(), res.cause()));
```